### DISCUSSION TITLE
manually select content to be downloaded from a magnet/torrent file

### DISCUSSION THREAD

**Discussion by st-ivan** [rogerfar/rdt-client#491] [Ideas]
Alldebrid user here.

Is there a way to manually select content to be downloaded from a magnet/torrent file? I think this is a useful feature to save both bandwidth and storage when downloading large torrents/magnets.

Is it possible to implement or is it supported already? thanks/

**Comment by rogerfar** [comment]
Unfortunately no, their API doesn't give an option to see which files are available and then to select specific files.
